src/core/index.ts

```diff
@@ -1,2 +1,4 @@
 export * from './Button';
+export * from './Tabs';
 export * from './Pills';
+export * from './TabPanel';
```

Comment:
I guess TabPanel is internally used for `Tabs` component and we shouldn't expose this

Reply:
@S-ayanide 
@arkajyotiMukherjee told me to make TabPanel as a separate component. Do I need to make it a separate one but in the Tabs folder?

Reply:
I am not sure of the use case for making this into a separate component, @arkajyotiMukherjee is there any advantage/scope for this to be extracted out locally?

Reply:
@saten16 you can just use 

```
<Tabs value={value} onChange={handleChange} aria-label="simple tabs example">
    <Tab label="Item One" {...a11yProps(0)} />
    <Tab label="Item Two" {...a11yProps(1)} />
    <Tab label="Item Three" {...a11yProps(2)} />
  </Tabs>
```

And remove the redundant components

Reply:
@S-ayanide It's exactly in this way now, just ina  dynamic way so that we can add label and content. 
Tell me please what do I supposed to do with TabPanel, should it be moved to Tabs folder or keep it where it is now (out of Tabs) ?

Reply:
You can keep it inside Tab.tsx only. Simply like its in MUI